tests/disable/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -0,0 +1,90 @@
+exports[`test disable.js 1`] = `
+"function a() {
+  // prettier-ignore-next-line
```

Comment:
I first named it ignore and then looked at eslint where it's disable. I must have forgotten to change this here :)